aria/modeling/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
These methods have almost no use on their own. They are utility methods that are exactly only useful for the complete instantiation of the service template. I suggest we wait until we have an instantiation (or however we will call it) module.